outlines/code-style.md

```diff
@@ -0,0 +1,22 @@
+# Code style
+
+1. JavaScript and ES2015
+  1. Use JavaScript and compile all of your code with the `ecmascript` package
+  1. Follow the Meteor style guide, based on the AirBnB style guide
+  2. Use ESLint using the standard config, which is made to work with `ecmascript`
```

Comment:
Link to blog posts about how to set up your linter with your code editor, talk through how to use it during development otherwise.

Consider adding something to your CI hook?